projectfiles/Xcode/The Battle for Wesnoth.xcodeproj/project.pbxproj

```diff
@@ -4240,8 +4234,6 @@
 				46DF5BCC1F46173700BE6D24 /* irdya_datetime.cpp */,
 				46DF5BCB1F46173700BE6D24 /* irdya_datetime.hpp */,
 				911F471B1CAE5A7E00F47035 /* iterable_pair.hpp */,
-				EC53B04D1B23BB0E002F758F /* make_enum.cpp */,
-				91B621931B76720B00B00E0F /* make_enum.hpp */,
```

Comment:
Every IDE file that had `make_enum.hpp` removed should probably have `enum_base.hpp` added.

Reply:
Missing a header won't break compilation, at least.

Reply:
For xcode in particular I'm also not sure what the hash (I'm assuming it's a hash) represents, so I don't actually know how to manually add those.

Reply:
Yeah, headers should be added, but it's not a compile error if they're missing. (I think it might break the Counterparts popup to switch between associated source and header files.)